internal/parser/parser_test.go

```diff
@@ -9812,6 +9812,97 @@ func TestSingleStatementParse(t *testing.T) {
 				},
 			},
 		},
+		{
+			"COMMIT with comment",
+			"COMMIT -- making a commit",
+			&ast.SQLStmt{
+				CommitStmt: &ast.CommitStmt{
+					Commit: token.New(1, 1, 0, 6, token.KeywordCommit, "COMMIT"),
+				},
+			},
+		},
+		{
+			`Commented FROM clause in SELECT`,
+			"SELECT /** FROM*/ users",
```

Comment:
You are not testing from `/*` to end of input, or did I miss it?

Reply:
Trailing comment line no. [9888](https://github.com/xqueries/xdb/pull/46/files#diff-58edabd268c2fcdc59e6065c182a8296R9888) `"SELECT 2 /* trailing comment"`

Reply:
missed it